framework/wazuh/vulnerability.py

```diff
@@ -83,3 +83,30 @@ def last_scan(agent_list):
     result.total_affected_items = data['totalItems']
 
     return result
+
+
+@expose_resources(actions=["vulnerability:read"], resources=["agent:id:{agent_list}"], post_proc_func=None)
+def get_inventory_summary(agent_list=None, field='', limit=common.database_limit):
```

Comment:
I think limit is not being applied properly for this endpoint. The summary should be always calculated using all elements in the db, but only the first `limit` number of values should be shown in the response.

Reply:
It is, since the query is using `GROUP BY` and then applying its `LIMIT`. An example of the query using `field=severity` is the following:

```sql
select severity, count(*) as total from vuln_cves group by severity order by total desc limit 1;
```

When we do not apply limit, this is the response using the dev environment:

```json
{
  "data": {
    "severity": {
      "High": 2,
      "Medium": 1,
      "Low": 1,
      "Critical": 1
    }
  },
  "error": 0
}
```

Adding `limit=1`, this is the response:

```json
{
  "data": {
    "severity": {
      "High": 2
    }
  },
  "error": 0
}
```